src/io/flutter/actions/DeviceSelectorAction.java

```diff
@@ -135,9 +139,12 @@ private void updateActions(@NotNull Project project, Presentation presentation)
     presentation.setText(null);
   }
 
+  // It's not clear if we need TransparentUpdate, but apparently it will make the UI refresh
+  // the display more often?
```

Comment:
OK, yeah.  That's curious.

I wonder if @alexander-doroshko has any thoughts here?